README.md

```diff
@@ -104,18 +124,49 @@ BBLAYERS =+ "${RDKROOT}/meta-openwrt"
 BBLAYERS =+ "${RDKROOT}/meta-mmx"
 ```
 
-Add into `conf/local.conf`
+Build image
+```bash
+bitbake rdk-generic-broadband-image
 ```
-PREFERRED_VERSION_libubox = "git"
-PREFERRED_VERSION_ubus = "git"
 
-# meta-openwrt patch on busybox is outdated
-BBMASK_append += "|meta-openwrt/recipes-tweaks/busybox/busybox_%.bbappend"
+## HOWTO run
 
-IMAGE_INSTALL_append = " packagegroup-mmx"
+Check that U-Bus daemon run
+```bash
+ps aux | grep ubusd
 ```
 
-Build image
 ```bash
-bitbake rdk-generic-broadband-image
+rpcd -s /var/run/ubus/ubus.sock -t 30 &
```

Comment:
Why isn't there a service file to start this?

Reply:
I've added systemd unit for rpcd